### PR TITLE
Test on Node 6

### PR DESCRIPTION
**Pull request by jamestalmage**

### CODE REVIEW

**Comment by sindresorhus** [comment]
AppVeyor doesn't yet have Node.js 6 for some reason. Seems like they manually package them instead of just relying on a version manager. Why am I not surprised...

**Comment by sindresorhus** [comment]
https://twitter.com/appveyor/status/725164672383590400

So we can just keep this open until then.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - '6'
   - '5'
```
Comment (sindresorhus) [inline .travis.yml]: Can you drop `5`?

> Node.js v5 will continue to be supported for the next two months in order to give developers currently using v5 time to transition to Node.js v6. - https://nodejs.org/en/blog/release/v6.0.0/

Comment (jamestalmage) [inline .travis.yml]: I was thinking of doing that in 2 months! 😄 

Seems I side with @novemberborn here.

**Comment by jamestalmage** [comment]
> AppVeyor doesn't yet have Node.js 6 for some reason
> ...
> So we can just keep this open until then.

Notice that and had the same conclusion

File: .travis.yml (hunk 1 above)
Comment (sindresorhus) [inline .travis.yml]: Ok, I don't feel strongly either way. Mostly just annoying having to do this update twice :p

**Comment by kevva** [comment]
Node 6 should be available on AppVeyor now.

**Pull request merged**